runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/counters/NameContext.java

```diff
@@ -31,7 +31,10 @@ public abstract class NameContext {
    * systemName} and a {@code userName}.
    */
   public static NameContext create(
-      String stageName, String originalName, String systemName, String userName) {
+      String stageName,
```

Comment:
seems like all of these are @Nullable based upon functions below

system name is null in forStage method
possibly stageName is always non-null and could be modified in accessor?

Reply:
Ping, I realized this is just method and not members generated by autovalue below. But can stageName be non-null below?

Reply:
done